tests/foreman/ui/test_contenthost.py

```diff
@@ -135,11 +137,30 @@ def test_positive_end_to_end(session, default_location, module_repos_collection_
 
     :CaseImportance: Critical
     """
+    rhel_lifecycle_status = 'Unknown'
+    _distro = module_repos_collection_with_manifest.distro
+    if _distro == 'rhel7':
+        rhel_lifecycle_status = 'Approaching end of maintenance support'
+    elif _distro == 'rhel8':
+        rhel_lifecycle_status = 'Full support'
+
     result = vm.run(f'yum -y install {FAKE_1_CUSTOM_PACKAGE}')
     assert result.status == 0
     startdate = datetime.utcnow().strftime('%m/%d/%Y')
     with session:
         session.location.select(default_location.name)
+        session.organization.select(module_org.name)
+        # Use new host UI, check the status and RHEL Lifecycle in details
+        session.settings.update(f'name = {"new_hosts_page"}', 'Yes')
```

Comment:
I might rather use the `settings_update` fixture instead. You don't need to care about setting the value back to whatever it was before - the fixture does it for you after the test finishes.
Also, now if any of the asserts between the updates fail, your target_sat would stay switched to `new_hosts_page: Yes`, which may cause the tests executed after this one to fail.